### PR TITLE
[parameters] Update masses and add latex names for masses

### PR DESCRIPTION
**Pull request by mreboud**
Related to #171

### CODE REVIEW

File: eos/parameters/hadron-properties.yaml [hunk 1]
```diff
@@ -5,177 +5,233 @@ groups:
   # Pseudoscalar Meson Masses
   # {{{
   - title: 'Pseudoscalar Meson Masses'
-    description: ''
+    description: 'Last update: December 2020'
```
Comment (dvandyk) [inline eos/parameters/hadron-properties.yaml:8]: I would not do that. The web page display for which version of EOS this was done.

File: eos/parameters/hadron-properties.yaml [hunk 2]
```diff
@@ -5,177 +5,233 @@ groups:
...
-          comments: 'cf. [PDG2019]'
+          latex:   '$m_{B_s^*}$'
+          comments: 'cf. [PDG-Live], December 2020'
   # }}}
 
   # Vector Meson Masses
   # {{{
   - title: 'Vector Meson Masses'
-    description: ''
+    description: 'Last update: December 2020'
```
Comment (dvandyk) [inline eos/parameters/hadron-properties.yaml:104]: See above.

File: eos/parameters/hadron-properties.yaml [hunk 3]
```diff
@@ -5,177 +5,233 @@ groups:
...
+          max:     +5.32491
+          latex:   '$m_{B^*}$'
+          comments: 'cf. [PDG-Live], December 2020'
   # }}}
 
   # Baryon Masses
   # {{{
   - title: 'Baryon Masses'
-    description: ''
+    description: 'Last update: December 2020'
```
Comment (dvandyk) [inline eos/parameters/hadron-properties.yaml:200]: And above.

**Review by dvandyk — CHANGES_REQUESTED** [review]
Thanks Meril for updating these. However, this has caused a few test case failures. Those test case should be update to use the original value for the masses they depended upon.

Also, can you prefix the commit message with '[paramters] ' please?

File: eos/parameters/hadron-properties.yaml (hunk 1 above)
Comment (mreboud) [reply eos/parameters/hadron-properties.yaml:8]: I agree, I will remove this

File: eos/parameters/hadron-properties.yaml (hunk 2 above)
Comment (mreboud) [reply eos/parameters/hadron-properties.yaml:104]: I agree, I will remove this

File: eos/parameters/hadron-properties.yaml (hunk 3 above)
Comment (mreboud) [reply eos/parameters/hadron-properties.yaml:200]: I agree, I will remove this

**Comment by mreboud** [comment]
> Thanks Meril for updating these. However, this has caused a few test case failures. Those test case should be update to use the original value for the masses they depended upon.

Yes, the main purpose of the commit was to add charmonium resonances ; I thought I could update other parameters in the same commit but it is indeed more complex that this. Somehow the checks passed locally... I am investigating

> Also, can you prefix the commit message with '[paramters] ' please?

Sure I keep forgetting this.

**Pull request merged**